rest_framework/static/rest_framework/css/default.css

```diff
@@ -32,10 +32,6 @@ h2, h3 {
   margin-right: 1em;
 }
 
-ul.breadcrumb {
-  margin: 70px 0 0 0;
-}
-
 .breadcrumb li.active a {
```

Comment:
I'm not sure what you mean. I'm *guessing* your pointing out that default.css and bootstrap-tweaks.css share a similar purpose and could be merged into a single file.